certbot/cli.py

```diff
@@ -1103,8 +1104,9 @@ def _create_subparsers(helpful):
     helpful.add(
         None, "--user-agent", default=None,
         help="Set a custom user agent string for the client. User agent strings allow "
-             "the CA to collect high level statistics about success rates by OS and "
-             "plugin. If you wish to hide your server OS version from the Let's "
+             "the CA to collect high level statistics about success rates by OS, "
+             "plugin and use case, and to know when to deprecate support for past Python "
+             "versions. If you wish to hide this information from the Let's "
```

Comment:
Let's Encrypt shares statistical data with EFF to help us improve Certbot and boulder, so in practice it's not inaccurate.